firebase-vscode/src/configs.ts

```diff
@@ -0,0 +1,144 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as vscode from "vscode";
+import { workspace } from "vscode";
+import { ExtensionBrokerImpl } from "./extension-broker";
+import { updateOptions, currentOptions } from "./options";
+import { RC } from "../../src/rc";
+import { Config } from "../../src/config";
+import { pluginLogger } from "./logger-wrapper";
```

Comment:
nit: to distance from the existing "Config.ts" can we change the name here to configUtils or configParsers or the like?

Reply:
You mean change the name of this file? Done.